### ISSUE TITLE
Repo layout

### ISSUE THREAD

**Issue by Phyks** [mapkyca/KnownGithub#3]
Hi,

What do you think about adopting the same layout as [IndiePub plugin](https://github.com/idno/indiepub)? (ie moving all the `Github/` content to the root)

This way, the plugin could be cloned very easily directly inside the `IdnoPlugins` folder and work out of the box (which will ease deployment and update thanks to git)

**Comment by mapkyca** [comment]
<a href="" class="u-in-reply-to"></a><a href="https://github.com/mapkyca/KnownGithub/issues/3" class="u-in-reply-to"></a>The issue is that the plugin requires the plugin to be deployed in a case sensitive directory "Github"... which is obviously not what the repo is called.

Yes, you can clone to a specific name, but in experience most don't, and it lends itself to confusion.

I'm actually having a think about a better way of loading plugins so hopefully this won't be a problem forever.

Myself, I deploy to an ext. directory and then symlink the dir... works quite well.

**Comment by Phyks** [comment]
Indeed, I had this case sensitivity issue with the IndiePub plugin… Maybe instructions with the correct command line to clone to the right folder in the `README.md` might be enough? Or about the symlink tip? (I had not thought about it, but will do it, it's a nice hack :)

EDIT: Same thing, "perform an oauth handshake" could be "view the callback URL in your browser and associate your Github account" for easier installation.
